asserts/system_user.go

```diff
@@ -106,6 +107,19 @@ func (su *SystemUser) Until() time.Time {
 	return su.until
 }
 
+// UserExpiration returns the expiration or validity duration of the user created.
+//
+// If no expiration was specified, this will return an empty time.Time structure.
```

Comment:
s/empty/zero/  as that will have IsZero true

Reply:
Ah nice catch, fixed!